cmd/arena/commands/get_advanced.go

```diff
@@ -65,39 +65,45 @@ func isTrainingConfigExist(name, trainingType, namespace string) bool {
 /**
 * BuildTrainingJobInfo returns types.TrainingJobInfo
  */
-func BuildJobInfo(job TrainingJob) *types.JobInfo {
+func BuildJobInfo(job TrainingJob) *types.TrainingJobPrinterInfo {
```

Comment:
Could you make the comment and method name consistent? Why using TrainingJobPrinterInfo to replace TrainingJob?